sotodlib/calibration/calibration_tod_analysis.py

```diff
@@ -0,0 +1,749 @@
+#!/usr/bin/env python3
+import so3g
+from so3g.proj import coords, quat
+import sotodlib.coords.planets as planets
+from sotodlib import core, tod_ops
+from sotodlib.tod_ops import filters
+from sotodlib.io.load import load_file
+from sotodlib.core import Context
+from sotodlib.toast import sim_sso
+
+import h5py
+import tables
+import numpy as np
+import scipy
+import scipy.stats
+import scipy.signal
+import time, ephem, copy, os
+from scipy import interpolate
+from scipy.optimize import curve_fit
+from glob import glob
+from datetime import datetime
+import argparse as ap
+import h5py
+import pandas as pd
+from mpi4py import MPI
+import healpy as hp
+
+opj  = os.path.join
+INITIAL_PARA_FILE = 'data/initial_parameters.hdf5'
+
+def highpass_filter(data, cutoff, fs, order=5):
```

Comment:
To the best of my recollection, I did this because the order of the Fourier filters was hard-coded in tod_ops.filters but it doesn't really make a significant difference. I think it was also in an experimenting phase when I wanted to check  whether/how much the optimal cutoff frequency fluctuates across different detectors of the focal plane ergo these (faster) mini filtering functions.  We can just delete those and use the sotodlib filters.

Reply:
Nadia, would you like to change this part of the code to the tod_ops.filters functions?